lib/index.js

```diff
@@ -756,7 +756,12 @@ class Board extends Emitter {
   servoWrite(pin, value) {
     const state = priv.get(this);
     const index = ToPinIndex(pin);
-    const constrained = constrain(value, 0, 180);
+    
+    if (value < 544) {
```

Comment:
Let's change this to 360 across all IO Plugins, as it will allow a wider range while being safe for standard servos that are capable of 360˚ (https://www.amazon.com/dp/B014L5CBBA ?)

Reply:
The pwmRange doesn't vary so we could only correctly scale the constrained value if the IO plugin knows the deviceRange. We could add that but we would be introducing something that doesn't exist in firmata.

Reply:
Ah, ok. Ignore my suggestion